pkg/webhook/validatingwebhook/validate_test.go

```diff
@@ -234,6 +258,204 @@ var sandboxUserJSON = []byte(`{
 	}
 }`)
 
+var sandboxUserJSON2 = []byte(`{
```

Comment:
Something like that:
```suggestion
var sandboxUserForAllServiceAccountsJSON = []byte(`{
```
would be more informative that JSON1, JSON2, etc.

Reply:
done